### PR TITLE
Splitting the common helper + small refactor

### LINKED ISSUE #146: The common.js file is getting too big
This is kind of a suggestion, the common.js file is getting bigger and we are just getting started.
It would be a good idea to introduce a controllers folder (following a typical MVC pattern), where we divide the code into a specific controller based on what it does. Something like auth controller, cart controller, products controller etc.

### PR DESCRIPTION
**Pull request by mrvautin**
Splitting the common helper into manageable and logical chunks - #146.

### CODE REVIEW

File: lib/cart.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    req.session.customerFirstname = null;
+    req.session.customerLastname = null;
+    req.session.customerAddress1 = null;
+    req.session.customerAddress2 = null;
+    req.session.customerCountry = null;
+    req.session.customerState = null;
+    req.session.customerPostcode = null;
+    req.session.customerPhone = null;
+    req.session.orderComment = null;
+};
```
Comment (happyhardik) [inline lib/cart.js:125]: This method seems odd in here, is it used by the cart and only by the cart?

File: lib/db.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ function initDb(dbUrl, callback){ // eslint-disable-line
     MongoClient.connect(dbUrl, { useNewUrlParser: true, useUnifiedTopology: true }, connected);
     function connected(err, client){
         if(err){
+            console.log('Failed connecting to the DB', err);
```
Comment (happyhardik) [inline lib/db.js:14]: Was this left by mistake, or there is a use of it in production?

**Review by happyhardik — APPROVED** [review]
Looks good to me!
Just a few small comments, but that should be fine, we can merge this.

File: lib/db.js (hunk 2 above)
Comment (mrvautin) [reply lib/db.js:14]: I added deliberately as the app fails silently when a DB connection cannot be made.

File: lib/cart.js (hunk 1 above)
Comment (mrvautin) [reply lib/cart.js:125]: This is used by the `order` and the `customer` route so I felt moving/keeping in a helper/lib file would be the best option.

**Pull request merged**